cmd/opm/alpha/bundle/build.go

```diff
@@ -69,11 +70,13 @@ func newBundleBuildCmd() *cobra.Command {
 
 	bundleBuildCmd.Flags().BoolVarP(&overwriteArgs, "overwrite", "o", false, "To overwrite annotations.yaml locally if existed. By default, overwrite is set to `false`.")
 
+	bundleBuildCmd.Flags().StringVarP(&outputDirArgs, "output-dir", "u", "", "Optional output directory for operator manifests")
+
 	return bundleBuildCmd
 }
 
 func buildFunc(cmd *cobra.Command, args []string) error {
-	err := bundle.BuildFunc(dirBuildArgs, tagBuildArgs, imageBuilderArgs,
+	err := bundle.BuildFunc(dirBuildArgs, outputDirArgs, tagBuildArgs, imageBuilderArgs,
```

Comment:
I avoided making any meaningful changes to this interface because it's currently implemented by the SDK team. Right now these functions are attempting to match the SDK's form and style as much as possible.

Personally, I would prefer to refactor this to pass a single struct that contains all parameters and a constructor that initiated a few defaults to allow for optional parameters given the complexity of the input. It just seems like we don't want to rock the boat too much with sdk adoption.

Reply:
I don't find variadic options is a good pattern for this case when those variables are used for different purposes. Some as dir, some for image tag and builder options. They should remain separate to avoid confusion. However, variadic options pattern does have the benefit that it may avoid future breaking API changes if we decide to add another string parameter into the func. Just need to handle the order.

Reply:
I actually like passing in a struct in cases like this, because it lets you just initialize the stuff you care about and check for zero values that you don't care about ex: https://github.com/operator-framework/operator-registry/blob/master/pkg/lib/indexer/indexer.go#L44

It lets you be hands off about requiring or ignoring certain parameters and lets the implementer make choices about it, which works pretty well when the indirection in front of the function is a CLI or library implementation